docs/semgrep-assistant/analyze.md

```diff
@@ -0,0 +1,45 @@
+---
+slug: analyze
+title: Analyze findings
+hide_title: true
+description: Learn how to analyze findings with Assistant and view its results.
+tags:
+  - Deployment
+  - Semgrep Assistant
+---
+
+# Analyze findings with Semgrep Assistant
+
+Once you've [enabled Assistant](/docs/semgrep-assistant/getting-started), you can use the **Analyze** button on the [Findings page](/semgrep-code/findings) to trigger all Assistant functions, including autofix, auto-triage, and component tagging, on existing findings.
+
+![Assistant Analyze button on Findings page](/img/scp-assistant.png#md-width)
+
+To analyze your findings with Assistant:
+
+1. On the [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page, select the findings that you want Assistant to analyze.
+2. Click **Analyze**.
+3. In the confirmation window that appears, confirm that you want to analyze your findings with Assistant.
+
+After Assistant performs these functions, you can see your results on the **Findings** page using the **Recommendation** or **Component** filters. When viewing your findings, you can see false positive and true positive recommendations when viewing the finding details pages if you choose **No Grouping** instead of **Group by Rule**.
```

Comment:
```suggestion
After Assistant performs these functions, you can see its results on the **Code** page using the **Recommendation** or **Component** filters. When viewing your findings, you can see false positive and true positive recommendations when viewing the finding details pages if you choose **No Grouping** instead of **Group by Rule**
```